packages/steiger/src/cli.ts

```diff
@@ -55,28 +72,22 @@ const yargsProgram = yargs(hideBin(process.argv))
   })
   .help('help', 'display help message')
   .alias('help', 'h')
-  .version()
+  .version(
+    [
+      packageJson.version,
+      $plugins
```

Comment:
We used to define and export getter functions from the config module before (like `getEnabledRules`, and `getGlobalIgnores`). Does it make sense in the case of plugins too for consistency?

Reply:
...so it just helps us to keep the codebase less dependent on the store solution and keep the code that interacts with effector mainly inside `models/config`

Reply:
I don't like getters for the sake of getters. In this case, it doesn't matter architecturally if we do `$plugins.getState()` or `getPlugins()`, because we aren't using Effector-specific functionality either way.

As for consistency, that is not the only store that's exported there, so we didn't have consistency there to begin with :)